lib/menu.rb

```diff
@@ -0,0 +1,15 @@
+class Menu
+  attr_reader :dishes
+  def initialize
+    @dishes = [
+        { name: 'Burger', price: 10.95  },
```

Comment:
Unnecessary spacing detected.<br>Use 2 spaces for indentation in an array, relative to the start of the line where the left bracket is.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.